### PR TITLE
Stream refactor

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
This refactor lifts the FrameTitlebar up to enable the fullscreen behaviour we're looking for in guides.

testable here: http://stream_refactor.surge.sh

### CODE REVIEW

File: src/browser/modules/Frame/FrameTitlebar.tsx [hunk 1]
```diff
@@ -127,138 +96,19 @@ type FrameTitleBarProps = FrameTitleBarBaseProps & {
 }
 
 function FrameTitlebar(props: FrameTitleBarProps) {
-  const [editorValue, setEditorValue] = useState(props.frame.cmd)
-  const [renderEditor, setRenderEditor] = useState(props.frame.isRerun)
+  const frame = props.frame
```
Comment (jharris4) [inline src/browser/modules/Frame/FrameTitlebar.tsx:99]: Any reason not to use destructuring here? ( `const { frame } = props` )

File: src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx [hunk 2]
```diff
@@ -102,9 +102,8 @@ function CypherScriptFrame({
       </ContentSizer>
     </WrapperCenter>
   )
-  return (
-    <FrameTemplate className="no-padding" header={frame} contents={contents} />
-  )
+  // This wanted to disable padding on the template. will have to do it in other way than classname.
```
Comment (jharris4) [inline src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx:105]: Looks like you've already addressed this in the StyledComponents logic, so this comment can be removed?

File: src/browser/modules/Stream/Stream.tsx [hunk 3]
```diff
@@ -50,9 +50,13 @@ import {
...
 import DbsFrame from './Auth/DbsFrame'
-import EditFrame from './EditFrame'
+import { StyledFrame } from '../Frame/styled'
+import FrameTitlebar from '../Frame/FrameTitlebar'
+import { dim } from 'browser-styles/constants'
+import styled from 'styled-components'
+import ExportButton, { ExportItem } from './ExportButtons'
 
-const trans = {
+const nameToFrame: Record<string, React.ComponentType<any>> = {
```
Comment (jharris4) [inline src/browser/modules/Stream/Stream.tsx:59]: Love that you've given this a name that's actually descriptive!

Comment (OskarDamkjaer) [reply src/browser/modules/Stream/Stream.tsx:59]: And it also ryhmes

File: src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx (hunk 2 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Stream/CypherScriptFrame/CypherScriptFrame.tsx:105]: yes, good catch

File: src/browser/modules/Frame/FrameTitlebar.tsx (hunk 1 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Frame/FrameTitlebar.tsx:99]: no

**Pull request closed without merging**